src/assets/sass/components/_charsheet.scss

```diff
@@ -0,0 +1,252 @@
+.charsheet {
+  &-stats {
+    @extend %cut-corners-box;
+
+    &__name {
```

Comment:
Why the double underscores?

Reply:
So this double underscores are actually something from BEM, a styling methodology/architecture to keep class specification in a lowest value possible.
Basically is a Block - Element - Modifier, where you set your block, like a style domain, the elements inside and then some possible modifier. With that we will make the code less dependent and more flexible as well!

Example:
.character-sheet-stats (block)
.character-sheet-stats__data-info (element)
.character-sheet-stats__data-info--edit (element modifier)


Here is their site if you wanna read more about this http://getbem.com/introduction/